### PR TITLE
fixing tooltip for see always part of the path name

### PR DESCRIPTION
**Pull request by anas-drissi**

### CODE REVIEW

File: src/components/study-container.js [hunk 1]
```diff
@@ -389,7 +389,8 @@ export function StudyContainer({ view, onChangeTab }) {
                     .slice(1)
                     .map((parent) => parent.elementName);
 
-                const studyName = response[0]?.elementName;
+                const studyName =
```
Comment (laurgarn) [inline src/components/study-container.js:392]: Why adding the ellipsis even when the name is short enough ?

File: src/components/study-container.js [hunk 2]
```diff
@@ -351,6 +351,10 @@ export function StudyContainer({ view, onChangeTab }) {
         // Note: studyUuid and dispatch don't change
     }, [studyUuid, dispatch, snackError, snackWarning]);
 
+    function limitChar(str) {
```
Comment (laurgarn) [inline src/components/study-container.js:354]: Would be better to define it out of the component function, as it avoids a new parsing of it at each call of the component function.

Comment (anas-drissi) [reply src/components/study-container.js]: it's fixed

**Review by laurgarn — APPROVED** [review]

File: src/components/study-container.js [hunk 3]
```diff
@@ -389,7 +393,7 @@ export function StudyContainer({ view, onChangeTab }) {
                     .slice(1)
                     .map((parent) => parent.elementName);
 
-                const studyName = response[0]?.elementName;
+                const studyName = limitChar(response[0]?.elementName, 30);
```
Comment (bendaamerahm) [inline src/components/study-container.js:396]: i think we don't do that here , there is a file that contain already a method computePageTitle in compute-title.js, just put your modification there

**Review by bendaamerahm — CHANGES_REQUESTED** [review]

**Review by bendaamerahm — APPROVED** [review]
code review: ok
test: ok

**Pull request merged**